solutions/swb-reference/package.json

```diff
@@ -20,6 +20,7 @@
     "cdk-deploy": "rush build && rushx compile && cdk deploy --all --require-approval never --outputs-file ./src/config/${STAGE}.json",
     "cdk-deploy:noBuild": "rushx compile && cdk deploy --all",
     "check-license-header": "license-check-and-add check -f license-add-config.json",
+    "cognito-tokens": "node scripts/generateCognitoTokens.js ${EMAIL} ${PASSWORD}",
```

Comment:
did you test this? you may need `${password}` in single quotes since sometimes there are special characters in the password

Reply:
like `"cognito-tokens": "node scripts/generateCognitoTokens.js ${EMAIL} '${PASSWORD}'"`

Reply:
Oh, if the user passes it as PASSWORD='6325^&*%^' that would work too.

Approving since this is related to the actual code here.